### DISCUSSION TITLE
Setting up site form template, questions on setup page

### DISCUSSION THREAD

**Discussion by Eaglef90** [OrchardCMS/OrchardCore#11565] [Q&A]
When I run the setup page from (https://docs.orchardcore.net/en/latest/docs/guides/decoupled-cms/#setting-up-a-new-site) I have two questions:

1.  I presume I don't want to be constantly logging in as the super user to create/edit my site so I would make another user but it would share my same personal email address as the super user. Can this be done?
2.  When I try to actually submit the page (hoping the answer to 1 is yes I can have another user with the same email) I get the error "Please match the requested format" in the password box. But no where does it let me choose a format or tell me what that is supposed to be. What format is this password supposed to be?

**Comment by Skrypt** [comment]
1. no
2. Format should be a valid email or maybe the issue here is that it returns the wrong error message intentionally. Need to see.

**Reply by Eaglef90** [reply]
Why does the super user password need to be in an email format? That is a very odd password requirement and if that is what is being forced on useres during inital setup that needs to be documented on the page somewhere.

**Reply by Skrypt** [reply]
I'm not sure what you are talking about I've never seen that issue. 

![image](https://user-images.githubusercontent.com/3228637/164331315-5b953573-3f77-47b8-809d-e216ab54c737.png)

**Reply by Eaglef90** [reply]
Here is a snip of the error. I see in your screen shot you don't even have a password in the field for the super user. I am wondering if you fill your out copleatly and submit it do you get the same error as me?

![error](https://user-images.githubusercontent.com/4780824/164462952-06c61280-6504-4a24-a072-f68ef1f44688.PNG)

**Reply by ns8482e** [reply]
It's browser's default validation - password requires specific regex pattern

**Reply by Eaglef90** [reply]
> It's browser's default validation - password requires specific regex pattern

Your right, I followed the typical capital,number, symbal, min 8 char requirements this time and it worked. Thanks you!
